src/main.rs

```diff
@@ -23,21 +25,22 @@ use rudr::schematic::{
 
 const DEFAULT_NAMESPACE: &str = "default";
 
-fn kubeconfig() -> kube::Result<kube::config::Configuration> {
+async fn kubeconfig() -> kube::Result<kube::config::Configuration> {
     // If env var is set, use in cluster config
     match std::env::var("KUBERNETES_PORT") {
         Ok(_val) => {
             info!("Loading in-cluster config");
```

Comment:
It seems it's blocked by some reason and always printing `Loading in-cluster config`.

Reply:
This log is from `sync_status`